src/interfaces/IERC20.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
```

Comment:
making some edits to the price oracle contract in a separate PR. since I'm already editing the other contract, if you back out these IERC20 changes, I can integrate them in the other PR. jsut don't want to have to deal with the conflict.